scrapenews/spiders/dailyvoice.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+from .sitemap import SitemapSpider
+from scrapenews.items import ScrapenewsItem
+from datetime import datetime
+import pytz
+
+SAST = pytz.timezone('Africa/Johannesburg')
+
+
+class DailyVoiceSpider(SitemapSpider):
+    name = 'dailyvoice'
+    allowed_domains = ['www.dailyvoice.co.za']
+
+    sitemap_urls = ['https://www.dailyvoice.co.za/robots.txt']
+    sitemap_follow = [
+        'www.dailyvoice.co.za/news/western-cape',
+        'www.dailyvoice.co.za/news/national',
+        'www.dailyvoice.co.za/news/politics',
+    ]
+
+    publication_name = 'Daily Voice'
+
+    def parse(self, response):
+        canonical_url = response.xpath('//link[@rel="canonical"]/@href').extract_first()
+        title = response.xpath('//h1/text()').extract_first()
+        self.logger.info('%s %s', response.url, title)
+        article_body = response.css('div.articleBodyMore')
+
+        if article_body:
+            body_html = article_body.extract_first()
+            byline = response.xpath('//strong[@itemprop="name"]/text()').extract_first()
+            publication_date_str = response.xpath('//span[@itemprop="datePublished"]/text()').extract_first()
```

Comment:
I think something like `//span[@itemprop="datePublished"]/@content` would give you the string in the content attribute of that element, which is already an ISO 8601 format date excluding timezone, e.g. `2018-06-08T07:22`. I don't have a strong argument for changing your code to use that, just thought I'd mention it if it turns out useful.